test/rules_test.js

```diff
@@ -205,6 +205,14 @@ describe('rules', () => {
         ['2x + 7y + 5 + 3y + 9x + 11', '(2 x + 9 x) + (7 y + 3 y) + (5 + 11)'],
     ])
 
+    suite('add polynomials', rules.ADD_POLYNOMIAL_TERMS, [
+        ['2x + 2x + 2 + 4', '4 x + (2 + 4)'],
+        ['2x + 2y', '2 x + 2 y'],
```

Comment:
This one fails b/c the pattern match returns false.  We should have better error handling for this, see https://github.com/semantic-math/math-rules/issues/27.